### PR TITLE
DEL: LocalTrader (HodlHodl) starts with Globabl offers by default (cl…

### PR DESCRIPTION
**Pull request by Overtorment**
…oses #2465)

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4mzryv8hfcctpndew64hd424wg

[download apk](http://filestorage.bluewallet.io:1488/fix-2465-1611690496.apk)

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/n7d3ygcvcnvxu00cag23206j58

[download apk](http://filestorage.bluewallet.io:1488/fix-2465-1611706527.apk)

File: package.json [hunk 1]
```diff
@@ -70,7 +70,6 @@
     "@react-native-async-storage/async-storage": "1.13.3",
     "@react-native-community/blur": "3.6.0",
     "@react-native-community/clipboard": "1.5.1",
-    "@react-native-community/geolocation": "2.0.2",
```
Comment (Overtorment) [inline package.json:73]: silly me, forgot to remove the module. gj @marcosrdz

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**